Core/include/Acts/EventData/VectorMultiTrajectory.hpp

```diff
@@ -47,6 +47,32 @@ using MultiTrajectoryTraits::IndexType;
 constexpr auto kInvalid = MultiTrajectoryTraits::kInvalid;
 constexpr auto MeasurementSizeMax = MultiTrajectoryTraits::MeasurementSizeMax;
 
+template <typename T>
+struct NonInitializingAllocator {
+  using value_type = T;
```

Comment:
nitpick: should this go in a utility or detail header?

Reply:
It's currently in the detail namespace for this file, that seemed sufficient from my point of view.